src/components/MainView/Layout.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+export default class Layout extends Component {
+
+  render() {
+    const { children } = this.props;
+    return (
+      <>
+        {children}
+      </>
+    );
+  }
+}
```

Comment:
What's the purpose of this file? I couldn't find a single time it is used

Reply:
We will need this file in the future in order to remove NavBar from MainView, Dashboard components and add NavBar just in one component. Now if I do this I should refresh the page so that buttons state changes. I'm investigating this question and will do this when I'll find the solution.